proofs.go

```diff
@@ -62,9 +62,9 @@ func (p *ProofU) Verify(pk *gabikeys.PublicKey, context, nonce *big.Int) bool {
 
 // correctResponseSizes checks the sizes of the elements in the ProofU proof.
 func (p *ProofU) correctResponseSizes(pk *gabikeys.PublicKey) bool {
+	minimum := big.NewInt(0)
 	maximum := new(big.Int).Lsh(big.NewInt(1), pk.Params.LvPrimeCommit+1)
 	maximum.Sub(maximum, big.NewInt(1))
-	minimum := new(big.Int).Neg(maximum)
 
 	return p.VPrimeResponse.Cmp(minimum) >= 0 && p.VPrimeResponse.Cmp(maximum) <= 0
```

Comment:
Wouldn't it be more natural to use `p.VPrimeResponse.Sign()` here instead of using a `minimum` bigint?

Reply:
I rather wouldn't cause it's very readable using minimum and maximum. I understand that it would save a bit of memory but I don't think it's worth it at cost of readability.